packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -246,13 +246,16 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
 
     function _evaluateElectionBatchBySimpleCounting() internal virtual {
         ElectionStore storage store = _electionStore();
-        ElectionData storage electionData = store.electionData;
+        ElectionData storage electionData = _electionData();
 
         uint maxBatchSize = store.maxProcessingBatchSize;
         uint previousBatchIdx = electionData.processedBatchIdx;
         uint lessVotedCandidateVotes;
         uint lessVotedCandidateIdx;
 
+        // set initial values
+        (lessVotedCandidateIdx, lessVotedCandidateVotes) = _findLessVoted(electionData.nextEpochRepresentativeVotes);
```

Comment:
I don't like the property name `nextEpochRepresentative*`. We're calling these 'members', so why use 'representatives'? Feels inconsistent.

Reply:
Fixed (closing also #609  )